common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/SignInCommandResult.kt

```diff
@@ -42,47 +41,78 @@ interface SignInCommandResult {
     data class Complete(
         override val correlationId: String,
         val authenticationResult: ILocalAuthenticationResult
-    ) :
-        SignInStartCommandResult, SignInWithContinuationTokenCommandResult, SignInSubmitCodeCommandResult,
-        SignInSubmitPasswordCommandResult
+    ) : SignInStartCommandResult, SignInWithContinuationTokenCommandResult,
+        SignInSubmitCodeCommandResult,
+        SignInSubmitPasswordCommandResult {
+        override fun toUnsanitizedString(): String = "Complete(correlationId=$correlationId)"
+
+        override fun containsPii(): Boolean = false
+
+        override fun toString(): String = toUnsanitizedString()
+    }
 
     data class PasswordRequired(
         override val correlationId: String,
         val continuationToken: String
-    ) :
-        SignInStartCommandResult
+    ) : SignInStartCommandResult {
+        override fun toUnsanitizedString(): String = "PasswordRequired(correlationId=$correlationId)"
+
+        override fun containsPii(): Boolean = false
+
+        override fun toString(): String = "PasswordRequired(correlationId=$correlationId)"
```

Comment:
How about following the practice `override fun toString(): String = toUnsanitizedString()`

Reply:
Updated